shaders/rgen.glsl

```diff
@@ -39,20 +39,30 @@ void main() {
     ivec2 blue_noise_coords = ivec2((gl_LaunchIDEXT.xy + ivec2(hash(t), hash(3 * t))) % blue_noise_size);
     vec4 random = imageLoad(blue_noise, blue_noise_coords);
 
+    // Todo: is there a better way to clear these in a raytracing pipeline...?
```

Comment:
I think vkCmdClearColorImage is what you're looking for, but this is fine.

Reply:
I think this is preferred, adding two extra writes to storage images is probably not ideal.